pkgs/development/libraries/gdal/default.nix

```diff
@@ -7,13 +7,13 @@ with lib;
 
 stdenv.mkDerivation rec {
   pname = "gdal";
-  version = "3.2.2";
+  version = "3.3.1";
 
   src = fetchFromGitHub {
     owner = "OSGeo";
     repo = "gdal";
-    rev = "a33784291d19015217ea2604988e53d448e14a07";
-    sha256 = "sha256-ynCju3chDfYtyrGmUE0n3kkaH2Mpm+/DDHHxCahjhSQ=";
+    rev = "f7f80b3a86507d623f5a43c6addb13e8408daa5c";
+    sha256 = "sha256:0832w40a92cl8ydkv0pk7nl1ivhfhbhqbhmpqmdjpi0126jlm2y0";
```

Comment:
```suggestion
    rev = "v{version}";
    sha256 = "0832w40a92cl8ydkv0pk7nl1ivhfhbhqbhmpqmdjpi0126jlm2y0";
```

Reply:
This suggestion is broken:

```
$ b gdal
source> trying https://github.com/OSGeo/gdal/archive/v{version}.tar.gz
source>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
source>                                  Dload  Upload   Total   Spent    Left  Speed
source> 100   120  100   120    0     0    383      0 --:--:-- --:--:-- --:--:--   382
source>   0    14    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
source> curl: (22) The requested URL returned error: 404
source> error: cannot download source from any mirror
error: builder for '/nix/store/pm636vqbg780r0qqz7fnmx9f70i5mpx5-source.drv' failed with exit code 1
error: 1 dependencies of derivation '/nix/store/8a339a1zg7b3wlwjklm69ck6jh7zg45l-gdal-3.3.1.drv' failed to build
```

Reply:
you need `"v${version}"` for the interpolation to work